src/libs/vpatterndb/vpiece.h

```diff
@@ -72,102 +72,109 @@ class VPointF;
-    int GetPassmarkNextSAPoints(const QVector<VPieceNode> &path, int index, const VSAPoint &passmarkSAPoint,
-                                const VContainer *data, VSAPoint &point) const;
-    bool GetSeamPassmarkSAPoint(const VSAPoint &previousSAPoint, const VSAPoint &passmarkSAPoint,
-                                const VSAPoint &nextSAPoint, const VContainer *data, QPointF &point) const;
-
-    bool IsPassmarkVisible(const QVector<VPieceNode> &path, int passmarkIndex) const;
-
-    QVector<QLineF> CreatePassmark(const QVector<VPieceNode> &path, int previousIndex, int passmarkIndex, int nextIndex,
-                                   const VContainer *data,
-                                   const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
-    QVector<QLineF> SAPassmark(const QVector<VPieceNode> &path, VSAPoint &previousSAPoint,
-                               const VSAPoint &passmarkSAPoint, VSAPoint &nextSAPoint, const VContainer *data,
-                               int passmarkIndex, const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
-    QVector<QLineF> BuiltInSAPassmark(const QVector<VPieceNode> &path, const VSAPoint &previousSAPoint,
-                                      const VSAPoint &passmarkSAPoint, const VSAPoint &nextSAPoint,
-                                      const VContainer *data, int passmarkIndex) const;
-
-    static int IsCSAStart(const QVector<CustomSARecord> &records, quint32 id);
+    int                      getNotchNextSAPoints(const QVector<VPieceNode> &path, int index,
+                                                     const VSAPoint &notchSAPoint,
+                                                     const VContainer *data, VSAPoint &point) const;
+    bool                     getSeamNotchSAPoint(const VSAPoint &previousSAPoint, const VSAPoint &notchSAPoint,
+                                                    const VSAPoint &nextSAPoint,
+                                                    const VContainer *data, QPointF &point) const;
+
+    bool                     isNotchVisible(const QVector<VPieceNode> &path, int notchIndex) const;
+
+    QVector<QLineF>          createNotch(const QVector<VPieceNode> &path, int previousIndex, int notchIndex,
+                                            int nextIndex, const VContainer *data,
+                                            const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
+    QVector<QLineF>          createSaNotch(const QVector<VPieceNode> &path, VSAPoint &previousSAPoint,
+                                        const VSAPoint &notchSAPoint, VSAPoint &nextSAPoint, const VContainer *data,
+                                        int notchIndex,
+                                        const QVector<QPointF> &seamAllowance = QVector<QPointF>()) const;
+    QVector<QLineF>          createBuiltInSaNotch(const QVector<VPieceNode> &path, const VSAPoint &previousSAPoint,
```

Comment:
createBuiltInSANotch

Reply:
Ditto as above... probably should be createBuiltInSeamAllowanceNotch().